src/models/lazy_load.rs

```diff
@@ -365,6 +365,9 @@ impl<T: Clone + 'static> LazyItem<T> {
 }
 
 impl<T: Clone + CustomSerialize + 'static> LazyItem<T> {
+    // @TODO: This function uses the `load_item` method of
```

Comment:
What do you think about this?

Reply:
Good catch - we should be using the `get_object` method here for caching purposes. However, there's another issue in this code that I just realized. By using the `load_item` method, we're directly deserializing `T`. What we should be doing instead is first deserialize the offsets of `T` and `versions`, and then deserialize both `T` and `versions` from their respective offsets. The `get_object` method uses the [lazy_item_deserialize_impl](https://github.com/cosdata/cosdata/blob/main/src/models/serializer/lazy_item.rs#L203) function which handles this correctly. Please update the code accordingly.

Reply:
I've pushed a commit - https://github.com/cosdata/cosdata/pull/132/commits/647e13759c5e854aae8987975b1ec00062303a6d. Please check.

Reply:
Please hold on. A fix is needed post rebase.

Reply:
Fixed.